lib/apis/v3/felixconfig.go

```diff
@@ -237,6 +237,13 @@ type FelixConfigurationSpec struct {
 	// leaving the kernel to choose the source address used.
 	DeviceRouteSourceAddress string `json:"deviceRouteSourceAddress,omitempty"`
 
+	// This defines the route protocol added to programmed device routes, by default this will be RTPROT_BOOT
+	// when left blank.
+	DeviceRouteProtocol *int `json:"deviceRouteProtocol;omitempty"`
```

Comment:
should us comma rather than semi-colon delimiter.  same below.

Reply:
Oh yes, now I see. Thank you.